Fabric.Identity.API/Services/CouchDbAccessService.cs

```diff
@@ -22,7 +23,12 @@ public class CouchDbAccessService : IDocumentDbService
 
         private string GetFullDocumentId<T>(string documentId)
         {
-            return $"{typeof(T).Name.ToLowerInvariant()}:{documentId}";
+            return ReplaceInvalidChars($"{typeof(T).Name.ToLowerInvariant()}:{documentId}");

```

Comment:
Should this method just be called on the "documentId" portion of the key, or do we want to include the Type Name as well?

Reply:
Yeah i see your point. While it doesn't hurt anything to do it on the full string including the type, the type will never contain an invalid character so there will be no replacement.